torch/_dynamo/utils.py

```diff
@@ -906,6 +908,20 @@ def add_compilation_metrics_to_chromium(c: Dict[str, Any]) -> None:
         dynamo_config=c["dynamo_config"],
     )
 
+def scrubbed_inductor_config_for_logging() -> Dict[str, Any]:
+    configs_to_scrub_re = r'((^TYPE_CHECKING$)|(.*_progress$)|(.*TESTING.*)|(.*(rocm|halide).*)|(^trace\..*)|(^_))'
+    keys_to_scrub = set()
+    inductor_config_copy = inductor_config.get_config_copy()
+    for key, val in inductor_config_copy.items():
+        if re.search(configs_to_scrub_re, key):
+            keys_to_scrub.add(key)
+        if not isinstance(val, str|bool|int|float):
```

Comment:
Is this valid syntax?

Reply:
It is, but I'll change it to typing.get_args(Union[str,int,bool,float]) so it is more readable

Reply:
Meh, fixed it